ophyd/areadetector/docs.py

```diff
@@ -2680,7 +2680,11 @@
           Toggles AutoGain mode for EVT camera. This will have the camera automatically adjust the gain rather than using the 
           user adjustable Gain value.
         ''',
-    ),  # end of EVTDoc.html
+        ),  # end of EVTDoc.html
+
+    'EigerDoc.html' : dict(
```

Comment:
@klauer Was there a script to scrape the contents of this file out of the upstream docs?